components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/pom.xml

```diff
@@ -52,10 +52,12 @@
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>org.wso2.micro.integrator.management.apis.internal</Private-Package>
```

Comment:
Do we need to explicitly private package this?

Reply:
Removed the private package in 86d3c947baab9b88291b959d3c35e2d03699c888